### PR TITLE
JBIDE-23671 Move ws reddeer wrapper classes from jbosstools-integration-tests to jbosstools-webservices/test-framework

### PR DESCRIPTION
**Pull request by psrna**

### CODE REVIEW

**Comment by psrna** [comment]
depends on https://github.com/jbosstools/jbosstools-base/pull/552

**Comment by psrna** [comment]
testPR

**Comment by psrna** [comment]
testPR

File: test-framework/org.jboss.tools.ws.reddeer/src/org/jboss/tools/ws/reddeer/ui/preferences/JBossWSRuntimeItem.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.jboss.tools.ws.reddeer.ui.preferences;
```
Comment (jeffmaury) [inline test-framework/org.jboss.tools.ws.reddeer/src/org/jboss/tools/ws/reddeer/ui/preferences/JBossWSRuntimeItem.java]: Header is missing

Comment (psrna) [inline test-framework/org.jboss.tools.ws.reddeer/src/org/jboss/tools/ws/reddeer/ui/preferences/JBossWSRuntimeItem.java]: Thx! Is fixed now.

**Pull request merged**